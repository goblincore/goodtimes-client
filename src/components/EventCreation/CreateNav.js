import React from 'react';
import '../styles/CreateNav.css';
import { FaAngleDoubleRight, FaAngleDoubleLeft } from 'react-icons/lib/fa';

export default function CreateNav(props) {

  const emoji=[
    null,
    <span className="ap ap-european_castle"></span>,
    <span className="ap ap-spiral_calendar_pad"></span>,
    <span className="ap ap-fries"></span>,
    <span className="ap ap-dancers"></span>,
    <span className="ap ap-mag_right"></span>,
  ];

  const pageSteps=[
    null,
    'Step 1 of 5 : Enter Title, Location, Description' ,
    'Step 2 of 5 : Select Multiple Time & Date Options',
    'Step 3 of 5 : Choose Food Options',
    'Step 4 of 5 : Choose Activity Options',
    'Step 5 of 6 : Preview Event Survey',
    'Step 6 of 6 : Thank you!',
  ];

  let iconAdjust={
    fontSize: '13px',
    transform: 'translateY(-1px)'
  };

  return(
 
   
    <nav className='create-nav'>
      <div className="instructions">
        <h4> {pageSteps[props.pageNum]} {emoji[props.pageNum]} </h4> 
      </div>

      <div className="nav-buttons">
        <button type='button' onClick={() => props.prevPage()}> <FaAngleDoubleLeft style={iconAdjust} /> Previous</button>
        <button type='button' 
          onClick={() => props.saveAsDraft()}>
                  Save as Draft
        </button>
        {   props.pageNum === 1 ?
          <button  type='submit' form='createform' value="Submit">Next <FaAngleDoubleRight  style={iconAdjust}  /></button> :
          <button type='button' onClick={props.handleNextPage}>Next <FaAngleDoubleRight  style={iconAdjust} /></button>
        }
               
      </div>
    </nav>

  );

}