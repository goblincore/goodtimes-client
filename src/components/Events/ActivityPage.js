import React from 'react';
import { Redirect } from 'react-router-dom';
import { fetchCategories, fetchActivities } from '../../actions/Activities';
import moment from 'moment';
import { updateNewEventState } from '../../actions/New-Event';

import SelectActivity from './SelectActivity';
import WriteActivity from './WriteActivity';

export default class ActivitySelect extends React.Component {

  constructor(props){
    super(props);
    this.state = {
      display: 'none'
    };
  }
  componentDidMount(){
    // If user hasn't selected any dates, they must redirect back to DateSelectPage
    if (this.props.times.length <= 0) {
      localStorage.setItem('eventDraft', JSON.stringify(this.props.eventState));
      return window.location.reload();
    }
    this.props.dispatch(fetchCategories());

  }
  filterEvents(e){
    const times = this.props.times.sort();
    e.preventDefault();
    this.props.dispatch(fetchActivities(this.props.latitude, this.props.longitude,times[0],times[times.length-1], e.target.value));
  }

  deleteWhenClicked(e){
    const { activityOptions }  = this.props.eventState;
    const idOfActivityToDelete = e.target.id;
    const filteredActivities = activityOptions.filter((option) => option._id !== idOfActivityToDelete);
  this.props.dispatch(updateNewEventState({activityOptions: filteredActivities}));
  }
  render(){
    //If user hasn't selected dates yet, they can't get events
    if(this.props.times.length <= 0) {
      return (
        <Redirect to={{
          pathname: '/edit-draft',
          state: {pageCount: 2}
        }} />
      )
    }
    
    let optionDisplay;

    if(this.state.display === 'none'){
      optionDisplay = <div></div>;
    }
    else if(this.state.display === 'write'){
      optionDisplay =   
      <div>
        <WriteActivity 
          dispatch={this.props.dispatch} 
          eventState={this.props.eventState}
          prevPage={this.props.prevPage} 
          nextPage={this.props.nextPage}
          categories={this.props.categories}
          activities={this.props.activities}
          loading={this.props.loading}
          latitude={this.props.latitude}
          longitude={this.props.longitude}
          times={this.props.times}
        />
        <button 
          onClick={(e) => {
            const form = e.target.parentElement.firstChild;
            this.props.dispatch(updateNewEventState({
              activityOptions: [...this.props.eventState.activityOptions, {
                ebId: form.title.value, link: '#', description: form.description.value, title: form.title.value
              }]
            })
            );
            this.setState({display:'none'});
          }}
        >Save Event</button>
      </div>;
    }
    else if(this.state.display === 'choose'){
      optionDisplay = <SelectActivity 
        dispatch={this.props.dispatch} 
        eventState={this.props.eventState}
        prevPage={this.props.prevPage} 
        nextPage={this.props.nextPage}
        categories={this.props.categories}
        activities={this.props.activities}
        loading={this.props.loading}
        latitude={this.props.latitude}
        longitude={this.props.longitude}
        times={this.props.times}/>;
    }
    let selectedActivitiesDisplay;
    if ( this.props.eventState.activityOptions.length > 0){
      selectedActivitiesDisplay = this.props.eventState.activityOptions.map((activity,index) => { 
      if(!activity.description){  
        return (  <div key={index}>
          <a href={activity.link} target='blank'>{activity.title}</a>
         </div>
     )
           } else {

            return (  <div key={index}>
              <a href={activity.link} target='blank'>{activity.title}:</a>
               <p>{activity.description.length > 50 ? `${activity.description.slice(0,50)}...` : activity.description}</p>
           </div>
)


            }
            });      
    }

    return(
      <div>
        <h1>Let's do something!</h1>
        <p>You can choose from events in your area or create your own!</p>
        <button onClick={() => this.setState({display: 'choose'})}>Choose From List</button>
        <button onClick={() => this.setState({display: 'write'})}>Create My Own</button>
        <div>Event Choices{selectedActivitiesDisplay}</div>
        {optionDisplay}
        <button type='button' onClick={() => this.props.prevPage()}>
          {'<-'} Back
        </button>
        <button type='button' 
            onClick={() => this.props.saveAsDraft()}>
            Save as Draft
          </button>
        <button type='button' onClick={()=>this.props.nextPage()}>Next Page</button>
      </div>
    );
  }
}
