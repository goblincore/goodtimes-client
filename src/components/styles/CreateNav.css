.instructions h4{
    margin:0px;
    font-size:1.2em;
}

.instructions{
    float:left;
    padding:0px !important;
}

.nav-buttons{
    float:right;
}

.create-nav{
    padding-bottom:20px;
    padding-right:30px;
    padding-left:20px;
    height:80px;
}

@media only screen and (max-width: 450px) {
    .nav-buttons{
      float:left;
      margin-top:20px;
    }

    #create-form{
        padding-bottom:60px;
    }
   
}
   