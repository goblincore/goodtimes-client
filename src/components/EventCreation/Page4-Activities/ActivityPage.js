import React from 'react';
import { Redirect } from 'react-router-dom';
import { updateNewEventState } from '../../../actions/New-Event';
import '../../styles/ActivitySelect.css';
import CreateNav from '../CreateNav';
import SelectActivity from './SelectActivity';
import WriteActivity from './WriteActivity';
import { MdHighlightOff } from "react-icons/lib/md";



export  class ActivityPage extends React.Component {

  constructor(props){
    super(props);
    this.state = {
      display: 'choose'
    };
  }

  componentDidMount(){
    // If user hasn't selected any dates, they must redirect back to DateSelectPage
    if (this.props.times.length <= 0) {
      localStorage.setItem('eventDraft', JSON.stringify(this.props.eventState));
      return window.location.reload();
    }
  }


  deleteWhenClicked(e){
    const { activityOptions }  = this.props.eventState;
    let targetNode = e.target;
    while (targetNode.tagName !== 'svg') {
      targetNode = targetNode.parentNode;
    }
    const idOfActivityToDelete = targetNode.dataset.ebid;
    if (document.getElementById(idOfActivityToDelete)) {
      document.getElementById(idOfActivityToDelete).checked = false;
    }
    const filteredActivities = activityOptions.filter((option) => option.ebId !== idOfActivityToDelete);
    this.props.dispatch(updateNewEventState({activityOptions: filteredActivities}));
  }


  render(){
    //If user hasn't selected dates yet, they can't get events
    if(this.props.times.length <= 0) return <Redirect to={{pathname: '/edit-draft', state: {pageCount: 2}}} />

    
    let optionDisplay;

    if(this.state.display === 'none'){
      optionDisplay = <div></div>;
    }
    else if(this.state.display === 'write'){
      optionDisplay = (
        <div>
          <WriteActivity 
            dispatch={this.props.dispatch} 
            eventState={this.props.eventState}
            prevPage={this.props.prevPage} 
            nextPage={this.props.nextPage}
            categories={this.props.categories}
            activities={this.props.activities}
            loading={this.props.loading}
            latitude={this.props.latitude}
            longitude={this.props.longitude}
            times={this.props.times}
          />
          <button 
            onClick={(e) => {
              const form = e.target.parentElement.firstChild;

              //verify that the activity is not already in the New Event state
              if (this.props.eventState.activityOptions.find(act => act.ebId === form.title.value)) {
                return this.props.dispatch(updateNewEventState({errorMessage: 'You already created that activity option.'}));
              }

              this.props.dispatch(updateNewEventState({
                errorMessage: '',
                activityOptions: [...this.props.eventState.activityOptions, {
                  ebId: form.title.value, description: form.description.value, title: form.title.value
                }]
              })
              );
              form.reset();
              // this.setState({display:'none'});
            }}
          >Save Event</button>
        </div>
      )
    }
    else if(this.state.display === 'choose'){
      optionDisplay = <SelectActivity 
        dispatch={this.props.dispatch} 
        eventState={this.props.eventState}
        prevPage={this.props.prevPage} 
        nextPage={this.props.nextPage}
        categories={this.props.categories}
        activities={this.props.activities}
        loading={this.props.loading}
        latitude={this.props.latitude}
        longitude={this.props.longitude}
        times={this.props.times}/>;
    }

    let selectedActivitiesDisplay;
    if ( this.props.eventState.activityOptions.length > 0){
      selectedActivitiesDisplay = this.props.eventState.activityOptions.map( activity => {
        let linkDisplay;
        let descriptionDisplay;
        // If theres an activity description
        if(!activity.description){
          descriptionDisplay = <p></p>;
        } else {
          descriptionDisplay =  <p>{activity.description.length > 100 ? `${activity.description.slice(0,100)}...` : activity.description}</p>;
        }

        // If theres an activity link
        if(!activity.link){
          linkDisplay = <p></p>;
        } else {
          linkDisplay = <span><a href={activity.link} target='_blank'>Go to event webpage.</a></span>;
         
        }

        return (  <div className="border-bottom" key={activity.ebId}>
         
          <p data-ebid={activity.ebId}>
            <MdHighlightOff  data-ebid={activity.ebId}  onClick={(e)=> this.deleteWhenClicked(e)} className="icon-adjust delete-icon"/>
            {activity.title}  
          </p>
          {descriptionDisplay}
          {linkDisplay}
        </div>
        );
      }
      );      
    }

    return(

      <div>

          <CreateNav saveAsDraft={this.props.saveAsDraft} pageNum={this.props.pageNum} prevPage={this.props.prevPage} nextPage={this.props.nextPage} handleNextPage={this.props.nextPage} />
    
          <div className="card border-right"> 
            <p className='form-error'>{this.props.eventState.errorMessage}</p>
            <button className="choose-from-list" onClick={() => this.setState({display: 'choose'})}>Choose From List</button>
            <button onClick={() => this.setState({display: 'write'})}>Create My Own Activity</button>
          </div>
    

          <div className="card" >
                <div className="activity-option">
                  {optionDisplay}
                </div>
          </div>


          <div id="user-event-list" className="card small-text">
            <h4>Event Choices{selectedActivitiesDisplay}</h4>
          </div>  

      </div>
     
    );
  }
}
