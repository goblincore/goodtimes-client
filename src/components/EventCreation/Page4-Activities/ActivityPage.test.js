import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow'; 
import {shallow, mount} from 'enzyme';
import  { ActivityPage } from './ActivityPage';
import CreateNav from '../CreateNav';
import renderer from 'react-test-renderer';

function setup() {
    const props = {
      activities: [{ url: 'google.com', name:'weekend at bernies', description:'laundry party',start: {
        local:
        "2018-10-11T18:00:00"
      }, end: {
        local:
        "2018-11-11T18:00:00"
      }}, 
      { url: 'google.com', name:'weekend at bernies', description:'mow my lawn as a gift' ,start: {
        local:
        "2018-10-11T18:00:00"
      }, end: {
        local:
        "2018-11-11T18:00:00"
      }}, 
      { url: 'google.com', name:'weekend at bernies', description: 'midnight fiesta' ,start: {
        local:
        "2018-10-11T18:00:00"
      }, end: {
        local:
        "2018-11-11T18:00:00"
      }}],
      latitude: 39.9526, 
      longitude: -75.1652,
      loading: false,
      categories: [{name:'italian', id: "111111111111111111111112"}, 
      {name: 'concert', id: "111111111111111111111114"}, 
      {name:'waterslide park', id: "111111111111111111111113"}],
      prevPage: jest.fn(),
      nextPage: jest.fn(),
       times: ["2018-10-30T11:30:00",
"2018-10-30T15:30:00",
"2018-10-09T21:30:00"],
     eventState: {
      location:{
        latitude: jest.fn(),
        longitude: jest.fn()
      },
      restaurantOptions: jest.fn(),
      scheduleOptions: jest.fn(),
      
      activityOptions: [
        {
          "ebId": "41090701394",
          "link": "https://www.eventbrite.com/e/kids-crossfit-ages-4-to-17-tickets-41090701394?aff=ebapi",
          "title": "Kids CrossFit - Ages 4 to 17!",
          "description":"get swole!",
          "shortUrl":"http://bit.ly/2E7TvIU",
          "start": "",
          "end": "",
          "votes": 0
        },
        {
          "ebId":"49111123693",
          "link":"https://www.eventbrite.com/e/open-mat-jiu-jitsu-all-levels-tickets-49111123693?aff=ebapi",
          "title":"Open Mat Jiu Jitsu - ALL Levels",
          "description": "Hi - ya!",
          "start": "",
          "end": "",
          "votes":0
        }
      ],
     },

     restaurants: {
       yelpRestaurants: jest.fn()
     },
      newEvent: 
       {
        restaurants: {
          yelpRestaurants: jest.fn()
        }
      },
  

      dispatch: jest.fn()
    }
  
    
    const enzymeWrapper = mount(<ActivityPage {...props} />)
  
    return {
      props,
      enzymeWrapper
    }
  }
  



describe('ActivityPage', () => {

    it('should render without crashing', () => {
      const { enzymeWrapper } = setup();
     
      expect(enzymeWrapper.find('h4').exists()).toBe(true);
    });

    it(`displays event options when 'Choose From List' clicked`, () => {
      const {props} = setup();
      const { enzymeWrapper } = setup();
      const chooseEventButton = enzymeWrapper.find('button').at(3);
      chooseEventButton.simulate('click');
  expect(enzymeWrapper.state().display).toEqual('choose');
    });
describe('Create My Own Activity', () => {
      it(`displays create activity form when 'Create My Own Activity' clicked`, () => {
         const {props} = setup();
          const { enzymeWrapper } = setup();
           const createEventButton = enzymeWrapper.find('button').at(4);
            createEventButton.simulate('click');
         expect(enzymeWrapper.state().display).toEqual('write');
             });
        
});
    //SNAPSHOT TEST

    it('renders as expected', () => {
      const props = {
        activities: [{ url: 'google.com', name:'weekend at bernies', description:'laundry party',start: {
          local:
          "2018-10-11T18:00:00"
        }, end: {
          local:
          "2018-11-11T18:00:00"
        }}, 
        { url: 'google.com', name:'weekend at bernies', description:'mow my lawn as a gift' ,start: {
          local:
          "2018-10-11T18:00:00"
        }, end: {
          local:
          "2018-11-11T18:00:00"
        }}, 
        { url: 'google.com', name:'weekend at bernies', description: 'midnight fiesta' ,start: {
          local:
          "2018-10-11T18:00:00"
        }, end: {
          local:
          "2018-11-11T18:00:00"
        }}],
        latitude: 39.9526, 
        longitude: -75.1652,
        loading: false,
        categories: [{name:'italian', id: "111111111111111111111112"}, 
        {name: 'concert', id: "111111111111111111111114"}, 
        {name:'waterslide park', id: "111111111111111111111113"}],
        prevPage: jest.fn(),
        nextPage: jest.fn(),
         times: ["2018-10-30T11:30:00",
  "2018-10-30T15:30:00",
  "2018-10-09T21:30:00"],
       eventState: {
        location:{
          latitude: jest.fn(),
          longitude: jest.fn()
        },
        restaurantOptions: jest.fn(),
        scheduleOptions: jest.fn(),
        
        activityOptions: [
          {
            "ebId": "41090701394",
            "link": "https://www.eventbrite.com/e/kids-crossfit-ages-4-to-17-tickets-41090701394?aff=ebapi",
            "title": "Kids CrossFit - Ages 4 to 17!",
            "description":"get swole!",
            "shortUrl":"http://bit.ly/2E7TvIU",
            "start": "",
            "end": "",
            "votes": 0
          },
          {
            "ebId":"49111123693",
            "link":"https://www.eventbrite.com/e/open-mat-jiu-jitsu-all-levels-tickets-49111123693?aff=ebapi",
            "title":"Open Mat Jiu Jitsu - ALL Levels",
            "description": "Hi - ya!",
            "start": "",
            "end": "",
            "votes":0
          }
        ],
       },
  
       restaurants: {
         yelpRestaurants: jest.fn()
       },
        newEvent: 
         {
          restaurants: {
            yelpRestaurants: jest.fn()
          }
        },
    
  
        dispatch: jest.fn()
      }
      const tree = renderer
      .create(<ActivityPage {...props}/>)
      .toJSON();
     
      expect(tree).toMatchSnapshot();
    });

    it('always renders a `CreateNav`', () => {
      const {props} = setup();
      const { enzymeWrapper } = setup();
      expect(enzymeWrapper.find(CreateNav).length).toBe(1);
    });

   
});


