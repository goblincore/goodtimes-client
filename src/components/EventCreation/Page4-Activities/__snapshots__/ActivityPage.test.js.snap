// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityPage renders as expected 1`] = `
<div>
  <nav
    className="create-nav"
  >
    <div
      className="instructions"
    >
      <h4>
         
         
         
      </h4>
    </div>
    <div
      className="nav-buttons"
    >
      <button
        onClick={[Function]}
        type="button"
      >
         
        <svg
          fill="currentColor"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "color": undefined,
              "fontSize": "13px",
              "transform": "translateY(-1px)",
              "verticalAlign": "middle",
            }
          }
          viewBox="0 0 40 40"
          width="1em"
        >
          <g>
            <path
              d="m22 30.7q0 0.3-0.2 0.5l-1.1 1.1q-0.3 0.3-0.6 0.3t-0.5-0.3l-10.4-10.4q-0.2-0.2-0.2-0.5t0.2-0.5l10.4-10.4q0.3-0.2 0.5-0.2t0.6 0.2l1.1 1.1q0.2 0.3 0.2 0.5t-0.2 0.6l-8.8 8.7 8.8 8.8q0.2 0.2 0.2 0.5z m8.6 0q0 0.3-0.3 0.5l-1.1 1.1q-0.2 0.3-0.5 0.3t-0.5-0.3l-10.4-10.4q-0.2-0.2-0.2-0.5t0.2-0.5l10.4-10.4q0.2-0.2 0.5-0.2t0.5 0.2l1.1 1.1q0.3 0.3 0.3 0.5t-0.3 0.6l-8.7 8.7 8.7 8.8q0.3 0.2 0.3 0.5z"
            />
          </g>
        </svg>
         Previous
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        Save as Draft
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        Next 
        <svg
          fill="currentColor"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "color": undefined,
              "fontSize": "13px",
              "transform": "translateY(-1px)",
              "verticalAlign": "middle",
            }
          }
          viewBox="0 0 40 40"
          width="1em"
        >
          <g>
            <path
              d="m22.3 21.4q0 0.3-0.2 0.5l-10.4 10.4q-0.3 0.3-0.6 0.3t-0.5-0.3l-1.1-1.1q-0.2-0.2-0.2-0.5t0.2-0.5l8.8-8.8-8.8-8.7q-0.2-0.3-0.2-0.6t0.2-0.5l1.1-1.1q0.3-0.2 0.5-0.2t0.6 0.2l10.4 10.4q0.2 0.2 0.2 0.5z m8.6 0q0 0.3-0.3 0.5l-10.4 10.4q-0.2 0.3-0.5 0.3t-0.5-0.3l-1.1-1.1q-0.2-0.2-0.2-0.5t0.2-0.5l8.8-8.8-8.8-8.7q-0.2-0.3-0.2-0.6t0.2-0.5l1.1-1.1q0.2-0.2 0.5-0.2t0.5 0.2l10.4 10.4q0.3 0.2 0.3 0.5z"
            />
          </g>
        </svg>
      </button>
    </div>
  </nav>
  <div
    className="card border-right"
  >
    <p />
    <button
      className="choose-from-list"
      onClick={[Function]}
    >
      Choose From List
    </button>
    <button
      onClick={[Function]}
    >
      Create My Own Activity
    </button>
  </div>
  <div
    className="card"
  >
    <div
      className="activity-option"
    >
      <div
        className="category-select"
      >
        <p>
          Change the category to see a list of events in your area during the times you selected. Check off events to add them to your list of activity options. You can select multiple events!
        </p>
        <select
          onChange={[Function]}
        >
          <option
            id="choose-a-category"
          >
            Choose a category...
          </option>
          <option
            id="111111111111111111111112"
            value="111111111111111111111112"
          >
            italian
          </option>
          <option
            id="111111111111111111111114"
            value="111111111111111111111114"
          >
            concert
          </option>
          <option
            id="111111111111111111111113"
            value="111111111111111111111113"
          >
            waterslide park
          </option>
        </select>
        <ul>
          <li
            className="activity-item border-bottom checked-false"
          >
            <input
              data-description={undefined}
              data-end="Sun, Nov 11, 2018 6:00 PM"
              data-start="Thu, Oct 11, 2018 6:00 PM"
              defaultChecked={false}
              id={undefined}
              name={undefined}
              onChange={[Function]}
              type="checkbox"
              value="google.com"
            />
            <a
              href="google.com"
              target="_blank"
            />
            <p
              className="dates-text"
            >
              Start: 
              Thu, Oct 11, 2018 6:00 PM
            </p>
            <p
              className="dates-text"
            >
              End: 
              Sun, Nov 11, 2018 6:00 PM
            </p>
          </li>
          <li
            className="activity-item border-bottom checked-false"
          >
            <input
              data-description={undefined}
              data-end="Sun, Nov 11, 2018 6:00 PM"
              data-start="Thu, Oct 11, 2018 6:00 PM"
              defaultChecked={false}
              id={undefined}
              name={undefined}
              onChange={[Function]}
              type="checkbox"
              value="google.com"
            />
            <a
              href="google.com"
              target="_blank"
            />
            <p
              className="dates-text"
            >
              Start: 
              Thu, Oct 11, 2018 6:00 PM
            </p>
            <p
              className="dates-text"
            >
              End: 
              Sun, Nov 11, 2018 6:00 PM
            </p>
          </li>
          <li
            className="activity-item border-bottom checked-false"
          >
            <input
              data-description={undefined}
              data-end="Sun, Nov 11, 2018 6:00 PM"
              data-start="Thu, Oct 11, 2018 6:00 PM"
              defaultChecked={false}
              id={undefined}
              name={undefined}
              onChange={[Function]}
              type="checkbox"
              value="google.com"
            />
            <a
              href="google.com"
              target="_blank"
            />
            <p
              className="dates-text"
            >
              Start: 
              Thu, Oct 11, 2018 6:00 PM
            </p>
            <p
              className="dates-text"
            >
              End: 
              Sun, Nov 11, 2018 6:00 PM
            </p>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div
    className="card small-text"
    id="user-event-list"
  >
    <h4>
      Event Choices
      <div
        className="border-bottom"
      >
        <p
          data-ebid="41090701394"
          onClick={[Function]}
        >
          Kids CrossFit - Ages 4 to 17!
        </p>
        <p>
          get swole!
        </p>
        <a
          href="https://www.eventbrite.com/e/kids-crossfit-ages-4-to-17-tickets-41090701394?aff=ebapi"
          target="_blank"
        >
          Go to event webpage.
        </a>
      </div>
      <div
        className="border-bottom"
      >
        <p
          data-ebid="49111123693"
          onClick={[Function]}
        >
          Open Mat Jiu Jitsu - ALL Levels
        </p>
        <p>
          Hi - ya!
        </p>
        <a
          href="https://www.eventbrite.com/e/open-mat-jiu-jitsu-all-levels-tickets-49111123693?aff=ebapi"
          target="_blank"
        >
          Go to event webpage.
        </a>
      </div>
    </h4>
  </div>
</div>
`;
